flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["i386"]
```

Comment:
I think being explicit with `only-arches` will be better here (but feel free to skip this as it's nitpicking).